hack/proto/Dockerfile

```diff
@@ -75,8 +76,9 @@ RUN protoc \
   v2/*.proto
 
 # this is a hack - seemingly grpc-gateway-swagger-gen is sometimes generating titles when they should be descriptions
-RUN jq 'walk(if type == "object" and has("title") then .description = ([.title, .description] | map(values) | join ("\n")) | del(.title) else .  end)' v1/skaffold.swagger.json | sponge v1/skaffold.swagger.json
-RUN jq 'walk(if type == "object" and has("title") then .description = ([.title, .description] | map(values) | join ("\n")) | del(.title) else .  end)' v2/skaffold.swagger.json | sponge v2/skaffold.swagger.json
+RUN wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && chmod +x ./jq
```

Comment:
It shouldn't be this hard to find an image with relatively recent toolsets 😢 
I wonder if grpc-gateway-swagger-gen has fixed this issue, or if it's a configuration issue on our part.

Reply:
I'll be looking more into this while testing out the newer versions of `protoc-gen-openapiv2`